web/client/src/components/validation/TokenValidation.jsx

```diff
@@ -57,7 +57,7 @@ export default function ValidateToken() {
     // Sending the Request
     validateToken(requestOptions)
       .then(async (response) => {
-        if (response.data["OAuth2l Response"] === "0") {
+        if (response.data["OAuth2lReponse"] === "0") {
```

Comment:
We should change this to async/await instead of using callbacks for consistency.

Reply:
done!